readme.md

```diff
@@ -82,14 +105,28 @@ Once installation is working, take a look at the existing code to make sure you
 
 * **Your Answer:**
 
+The password is sent in plaintext, unencrypted, which can be intercepted and read.
+
+The username and password are packaged together, conveniently for a hacker.
+
+
 * **Question:** What would happen if three different users tried to sign up with the same username? How can we prevent that?
 
 * **Your Answer:**
 
+Search the database to see if the username exists, and if so, send message to the user to require the user to create a different username.
```

Comment:
Correct, Currently the app allows multiple users to register with identical usernames and passwords